arangod/Graph/Providers/ClusterProvider.cpp

```diff
@@ -262,15 +264,31 @@ void ClusterProvider<StepImpl>::destroyEngines() {
 }
 
 template<class StepImpl>
-Result ClusterProvider<StepImpl>::fetchEdgesFromEngines(
-    VertexType const& vertex) {
+Result ClusterProvider<StepImpl>::fetchEdgesFromEngines(Step* step) {
+  TRI_ASSERT(step != nullptr);
+
   auto const* engines = _opts.engines();
-  // TODO Assert that the vertex is not in _vertexConnections after no-loose-end
+  // TODO Assert that the step is not in _vertexConnections after no-loose-end
```

Comment:
Please handle this TODO, or at least clarify it, does not really make sense to me anymore